.env

```diff
@@ -33,6 +33,7 @@ DATABASE_URL=postgres://postgres:postgrespass@localhost:5432/postgres?schema=api
 S3_ENDPOINT=https://cdn-dev.podkrepi.bg
 S3_ACCESS_KEY=
 S3_SECRET_ACCESS_KEY=
+S3_BUCKET_BANK_TRANSACTION=banktransactions-files
```

Comment:
IMHO the bucket names shouldn't be specified with the environment variables, because they are the same concept like the tables of a database, which we don't put in .env file.

On the other hand, the S3 access is ok to be in the .env in the same way the Database server access is.

Reply:
Alright, closing the PR then